extensions/BugModal/template/en/default/bug_modal/create.html.tmpl

```diff
@@ -0,0 +1,812 @@
+[%# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/.
+  #
+  # This Source Code Form is "Incompatible With Secondary Licenses", as
+  # defined by the Mozilla Public License, v. 2.0.
+  #%]
+
+[%
+  USE Bugzilla;
+
+  bug_modal = 1;
+
+  # use the placeholder bug
+  bug = default;
+
+  # reset variables to avoid unexpected inheritance
+  default = undef;
```

Comment:
Thanks for the fix for assignee and qa_contact. Those and other user fields now pre-fill properly. Other fields form my earlier comment still do not fill but I think I see the issue. For the security groups being not being automatically checked if for example if we pass `groups=core-security-release&groups=mozilla-employee-confidential`  it is related to bug_modal/create.html.tmpl setting default = undef in the top of this file. Then bug/create/security.html.tmpl is looking for `default.groups.contains(group.name)` to see if the checkbox should be checked which of course is empty. This same file on the legacy form does work properly. I would imagine that any form fields in the new modal form that look for default.* would not see any values because of the undef. Converting to `bug.groups.contains(group.name)` will work for modal form but will break the checkboxes for the legacy form. A hack would be to not undef default and allow using either bug.* or default.* but maybe you have a better idea?

This might also fix the other fields that are not being pre-filled:
 * status whiteboard
  * qa whiteboard (cf_qa_whiteboard)
  * user story
  * crash signature
  * time tracking original estimate and hours left (deadline is pre-filled correctly)

Reply:
QA whiteboard (`cf_qa_whiteboard`) not being prefilled is an administration issue. Go to `/editfields.cgi?action=edit&name=cf_qa_whiteboard`, tick “Can be set on bug creation”, submit, and I see the field prefilled. It’s [fillered here](https://github.com/mozilla-bteam/bmo/blob/master/enter_bug.cgi#L244). The old form didn’t have the field, actually.

Reply:
- Status Whiteboard: Fixed!
- QA whiteboard: See above. Most of the custom fields, including Rank and flags are actually not prefilled due to the same admin issue. Open `/editfields.cgi` to see that almost all fields come with `0` on the “Editable on Bug Creation“ column.
- User Story: Fixed!
- Crash Signature: It was prefilled but wasn’t visible at first glance. The field now behaves like the old form: it shows a textbox with a value prefilled or hides the textbox if no prefill value is set.
- Time Tracking: Fixed! I have removed the Hours Worked and Hours Left fields which seem unnecessary and aren’t on the old form.